package.json

```diff
@@ -30,10 +30,10 @@
   },
   "devDependencies": {
     "ava": "^2.4.0",
-    "stylelint": "^11.0.0",
+    "stylelint": "11.0.0",
```

Comment:
I don't see any reason to pin the version here, please revert.

Reply:
My motivation in pinning this was to make it harder to accidentally introduce backwards-incompatible changes. I would always lock a devDependency corresponding to a peerDependency range at the lowest supported version.

This config has the peerDependency range set to 11.x; the lowest version of Stylelint that this config supports and uses features from is 11.0.0, and that's why it's the version that's tested. If, for example, this config was modified to use a feature from stylelint@11.1, then this config would no longer be compatible with codebases with the stylelint dependency locked at 11.0, so the peerDependency range would need bumping - but if the test was run using stylelint@11.1, then it wouldn't catch this. The yarn.lock file is currently fixing this devDependency to 11.0.0, but it's a bit disconcerting to me that running `yarn upgrade` could obscure away a backwards-incompatible change.

I'm not sure about the wisdom of this change, but I've pushed a commit unpinning this dependency as requested.